apps/idos-enclave/src/pages/App.tsx

```diff
@@ -55,7 +56,7 @@ function Layout({ onHeaderClick, children }: PropsWithChildren<{ onHeaderClick?:
 
 export function App({ store, enclave }: AppProps) {
   const [method, setMethod] = useState<Method | null>(null);
-  const [mode, setMode] = useState<Mode>("existing");
+  const [mode, setMode] = useState<Mode>(store.get("human-id") ? "existing" : "new");
```

Comment:
Hard to review this for me. Punting it over to @ditoglez. Please pay attention to how `mode` and `method` are interacting.